CITATION.cff

```diff
@@ -27,6 +27,8 @@ authors:
 - family-names: "Geda"
   given-names: "Robel"
   orcid: "https://orcid.org/0000-0003-1509-9966"
+- family-names: "Green"
+  given-names: "Gilbert"
```

Comment:
@gibsongreen - did you want to list an orcid?  Otherwise if you're happy with everything, feel free to approve and we'll get this in!

Reply:
@kecnry, sorry I meant to send the id sooner, thank you!

orcid: "https://orcid.org/0009-0004-2370-0699"

Reply:
Looks good to me!